src/components/AuthEvents/EditEvent.js

```diff
@@ -0,0 +1,148 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import ReactDOM from 'react-dom';
+import SelectField from 'material-ui/SelectField';
+import MenuItem from 'material-ui/MenuItem';
+import DatePicker from 'material-ui/DatePicker';
+
+//components
+import Navigation from '../Navigation/Navigation';
+import Footer from '../Footer/Footer';
+import Category from '../category/GetCategories';
+
+import { getEventDetail } from '../../actions/eventActions';
+import { editEventAction } from '../../actions/authEvents.actions';
+
+
+const initialState = {
+	title: '',
+	location: '',
+	time: '',
+	date: '',
+	description: '',
+	image_url:'',
+	event_category: null
+};
+
+
+class EditEvent extends Component{
+
+    constructor(props){
+		super(props);
+        this.state = {
+			eventData: initialState,
+		};
+		this.handleInputChange = this.handleInputChange.bind(this);
```

Comment:
@meshnesh  as @PaulKariukiRimiru has mentioned below, using arrow functions saves you the hustle of having to bind `this` in every method